src/mender-inventory-mender-configure

```diff
@@ -21,7 +21,12 @@ set -e
 exec 1>&2
 
 CONFIG=${TEST_CONFIG:-/var/lib/mender-configure/device-config.json}
-CONFIG_ENDPOINT=${TEST_CONFIG_ENDPOINT:-configuration}
+CONFIG_ENDPOINT=${TEST_CONFIG_ENDPOINT:-api/devices/v1/deviceconfig/configuration}
```

Comment:
You probably need to fix this as well: https://github.com/mendersoftware/mender-configure-module/blob/master/tests/helpers/mender-configure-server.py#L23